views/js/pciCreator/ims/likertScaleInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -0,0 +1,28 @@
+<div class="panel">
+    <label for="level">{{__ "Level"}}</label>
+    <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
+    <span class="tooltip-content">{{__ "Scale size"}}</span>
+    <select name="level" class="select2" data-has-search="false">
+        {{#each levels}}
+        <option value="{{@key}}" {{#if selected}}selected="selected"{{/if}}>{{label}}</option>
+        {{/each}}
+    </select>
+</div>
+
+<div class="panel">
+    <label>
+        <input name="icons" type="checkbox" {{#if icons}}checked="checked"{{/if}}/>
```

Comment:
When editing existing item in Authoring, checkboxes are always checked, even though they weren't checked when I created the item

https://github.com/oat-sa/extension-tao-itemqti-pci/assets/38751932/46bbeba2-0714-49b9-801c-43cc1849d6bd

Reply:
In **Firefox**!
Ok in Chrome.

Reply:
Thanks. Fixed in https://github.com/oat-sa/extension-tao-itemqti-pci/pull/394/commits/10f83418257b25e1e42a4678a9f55ce7bddd036a